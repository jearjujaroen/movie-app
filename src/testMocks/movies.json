{
  "page": 1,
  "total_results": 10000,
  "total_pages": 500,
  "results": [
    {
      "popularity": 609.311,
      "vote_count": 3636,
      "video": false,
      "poster_path": "/xBHvZcjRiWyobQ9kxBhO6B2dtRI.jpg",
      "id": 419704,
      "adult": false,
      "backdrop_path": "/5BwqwxMEjeFtdknRV792Svo0K1v.jpg",
      "original_language": "en",
      "original_title": "Ad Astra",
      "genre_ids": [18, 878],
      "title": "Ad Astra",
      "vote_average": 6,
      "overview": "The near future, a time when both hope and hardships drive humanity to look to the stars and beyond. While a mysterious phenomenon menaces to destroy life on planet Earth, astronaut Roy McBride undertakes a mission across the immensity of space and its many perils to uncover the truth about a lost expedition that decades before boldly faced emptiness and silence in search of the unknown.",
      "release_date": "2019-09-17"
    },
    {
      "popularity": 416.549,
      "vote_count": 31,
      "video": false,
      "poster_path": "/avl9MEQhtvokNnzoWepkmHBZ2ss.jpg",
      "id": 574982,
      "adult": false,
      "backdrop_path": "/4phMeITqEnShF5C3ZD0Q8YQFMAB.jpg",
      "original_language": "ru",
      "original_title": "The Blackout",
      "genre_ids": [28, 878, 53],
      "title": "The Blackout",
      "vote_average": 6,
      "overview": "No meteorites hit Earth, no terrorists put the world in danger, no atomic war was started, but something did go wrong. Contact between most towns on Earth has been severed. A small ring-like area in Eastern Europe still has electricity, and maybe even life is being reported from the Space. What military forces find outside the Ring is shocking. There are dead corpses everywhere: in stores, in cars, on roads, in hospitals and railway stations. Who or what is destroying all life on Earth? How long will the last outpost of mankind survive?",
      "release_date": "2019-11-21"
    },
    {
      "popularity": 163.863,
      "vote_count": 4439,
      "video": false,
      "poster_path": "/aQvJ5WPzZgYVDrxLX4R6cLJCEaQ.jpg",
      "id": 454626,
      "adult": false,
      "backdrop_path": "/ibMSQUtltvnmsOSrCsBrDNThzCu.jpg",
      "original_language": "en",
      "original_title": "Sonic the Hedgehog",
      "genre_ids": [28, 35, 878, 10751],
      "title": "Sonic the Hedgehog",
      "vote_average": 7.5,
      "overview": "Based on the global blockbuster videogame franchise from Sega, Sonic the Hedgehog tells the story of the world’s speediest hedgehog as he embraces his new home on Earth. In this live-action adventure comedy, Sonic and his new best friend team up to defend the planet from the evil genius Dr. Robotnik and his plans for world domination.",
      "release_date": "2020-02-12"
    },
    {
      "popularity": 122.055,
      "vote_count": 2537,
      "video": false,
      "poster_path": "/8WUVHemHFH2ZIP6NWkwlHWsyrEL.jpg",
      "id": 338762,
      "adult": false,
      "backdrop_path": "/ocUrMYbdjknu2TwzMHKT9PBBQRw.jpg",
      "original_language": "en",
      "original_title": "Bloodshot",
      "genre_ids": [28, 878],
      "title": "Bloodshot",
      "vote_average": 7,
      "overview": "After he and his wife are murdered, marine Ray Garrison is resurrected by a team of scientists. Enhanced with nanotechnology, he becomes a superhuman, biotech killing machine—'Bloodshot'. As Ray first trains with fellow super-soldiers, he cannot recall anything from his former life. But when his memories flood back and he remembers the man that killed both him and his wife, he breaks out of the facility to get revenge, only to discover that there's more to the conspiracy than he thought.",
      "release_date": "2020-03-05"
    },
    {
      "popularity": 124.024,
      "vote_count": 7772,
      "video": false,
      "poster_path": "/7IiTTgloJzvGI1TAYymCfbfl3vT.jpg",
      "id": 496243,
      "adult": false,
      "backdrop_path": "/ApiBzeaa95TNYliSbQ8pJv4Fje7.jpg",
      "original_language": "ko",
      "original_title": "기생충",
      "genre_ids": [35, 18, 53],
      "title": "Parasite",
      "vote_average": 8.5,
      "overview": "All unemployed, Ki-taek's family takes peculiar interest in the wealthy and glamorous Parks for their livelihood until they get entangled in an unexpected incident.",
      "release_date": "2019-05-30"
    },
    {
      "popularity": 100.849,
      "vote_count": 794,
      "video": false,
      "poster_path": "/wxPhn4ef1EAo5njxwBkAEVrlJJG.jpg",
      "id": 514847,
      "adult": false,
      "backdrop_path": "/naXUDz0VGK7aaPlEpsuYW8kNVsr.jpg",
      "original_language": "en",
      "original_title": "The Hunt",
      "genre_ids": [28, 27, 53],
      "title": "The Hunt",
      "vote_average": 6.7,
      "overview": "Twelve strangers wake up in a clearing. They don't know where they are—or how they got there. In the shadow of a dark internet conspiracy theory, ruthless elitists gather at a remote location to hunt humans for sport. But their master plan is about to be derailed when one of the hunted turns the tables on her pursuers.",
      "release_date": "2020-03-11"
    },
    {
      "popularity": 101.453,
      "vote_count": 7483,
      "video": false,
      "poster_path": "/4q2NNj4S5dG2RLF9CpXsej7yXl.jpg",
      "id": 429617,
      "adult": false,
      "backdrop_path": "/5myQbDzw3l8K9yofUXRJ4UTVgam.jpg",
      "original_language": "en",
      "original_title": "Spider-Man: Far from Home",
      "genre_ids": [28, 12, 878],
      "title": "Spider-Man: Far from Home",
      "vote_average": 7.6,
      "overview": "Peter Parker and his friends go on a summer trip to Europe. However, they will hardly be able to rest - Peter will have to agree to help Nick Fury uncover the mystery of creatures that cause natural disasters and destruction throughout the continent.",
      "release_date": "2019-06-28"
    },
    {
      "popularity": 98.403,
      "vote_count": 5277,
      "video": false,
      "poster_path": "/iZf0KyrE25z1sage4SYFLCCrMi9.jpg",
      "id": 530915,
      "adult": false,
      "backdrop_path": "/cqa3sa4c4jevgnEJwq3CMF8UfTG.jpg",
      "original_language": "en",
      "original_title": "1917",
      "genre_ids": [28, 18, 36, 10752],
      "title": "1917",
      "vote_average": 7.9,
      "overview": "At the height of the First World War, two young British soldiers must cross enemy territory and deliver a message that will stop a deadly attack on hundreds of soldiers.",
      "release_date": "2019-12-25"
    },
    {
      "popularity": 98.693,
      "vote_count": 2105,
      "video": false,
      "poster_path": "/f4aul3FyD3jv3v4bul1IrkWZvzq.jpg",
      "id": 508439,
      "adult": false,
      "backdrop_path": "/xFxk4vnirOtUxpOEWgA1MCRfy6J.jpg",
      "original_language": "en",
      "original_title": "Onward",
      "genre_ids": [12, 16, 35, 14, 10751],
      "title": "Onward",
      "vote_average": 7.9,
      "overview": "In a suburban fantasy world, two teenage elf brothers embark on an extraordinary quest to discover if there is still a little magic left out there.",
      "release_date": "2020-02-29"
    },
    {
      "popularity": 98.069,
      "vote_count": 4342,
      "video": false,
      "poster_path": "/h4VB6m0RwcicVEZvzftYZyKXs6K.jpg",
      "id": 495764,
      "adult": false,
      "backdrop_path": "/kvbbK2rLGSJh9rf6gg1i1iVLYQS.jpg",
      "original_language": "en",
      "original_title": "Birds of Prey (and the Fantabulous Emancipation of One Harley Quinn)",
      "genre_ids": [28, 35, 80],
      "title": "Birds of Prey (and the Fantabulous Emancipation of One Harley Quinn)",
      "vote_average": 7.2,
      "overview": "Harley Quinn joins forces with a singer, an assassin and a police detective to help a young girl who had a hit placed on her after she stole a rare diamond from a crime lord.",
      "release_date": "2020-02-05"
    },
    {
      "popularity": 93.441,
      "vote_count": 9456,
      "video": false,
      "poster_path": "/qa6HCwP4Z15l3hpsASz3auugEW6.jpg",
      "id": 920,
      "adult": false,
      "backdrop_path": "/sd4xN5xi8tKRPrJOWwNiZEile7f.jpg",
      "original_language": "en",
      "original_title": "Cars",
      "genre_ids": [12, 16, 35, 10751],
      "title": "Cars",
      "vote_average": 6.8,
      "overview": "Lightning McQueen, a hotshot rookie race car driven to succeed, discovers that life is about the journey, not the finish line, when he finds himself unexpectedly detoured in the sleepy Route 66 town of Radiator Springs. On route across the country to the big Piston Cup Championship in California to compete against two seasoned pros, McQueen gets to know the town's offbeat characters.",
      "release_date": "2006-06-08"
    },
    {
      "popularity": 92.519,
      "vote_count": 0,
      "video": false,
      "poster_path": "/shx7pGYQ9s4t7WFe19eES2WOrMS.jpg",
      "id": 610201,
      "adult": false,
      "backdrop_path": "/lvtEcdBmmrxzAQvRIYyDRQDeDJM.jpg",
      "original_language": "en",
      "original_title": "The Pale Door",
      "genre_ids": [27, 37],
      "title": "The Pale Door",
      "vote_average": 0,
      "overview": "After a stagecoach robbery goes bad, two brothers leading a gang of cowboys must survive the night in a ghost town inhabited by a coven of witches.",
      "release_date": "2020-06-18"
    },
    {
      "popularity": 79.754,
      "vote_count": 906,
      "video": false,
      "poster_path": "/jHo2M1OiH9Re33jYtUQdfzPeUkx.jpg",
      "id": 385103,
      "adult": false,
      "backdrop_path": "/fKtYXUhX5fxMxzQfyUcQW9Shik6.jpg",
      "original_language": "en",
      "original_title": "Scoob!",
      "genre_ids": [12, 16, 35, 9648, 10751],
      "title": "Scoob!",
      "vote_average": 8,
      "overview": "In Scooby-Doo’s greatest adventure yet, see the never-before told story of how lifelong friends Scooby and Shaggy first met and how they joined forces with young detectives Fred, Velma, and Daphne to form the famous Mystery Inc. Now, with hundreds of cases solved, Scooby and the gang face their biggest, toughest mystery ever: an evil plot to unleash the ghost dog Cerberus upon the world. As they race to stop this global “dogpocalypse,” the gang discovers that Scooby has a secret legacy and an epic destiny greater than anyone ever imagined.",
      "release_date": "2020-05-15"
    },
    {
      "popularity": 125.121,
      "vote_count": 120,
      "video": false,
      "poster_path": "/kizBwSEfvJFEOuYfwbOGtWZit3v.jpg",
      "id": 363111,
      "adult": false,
      "backdrop_path": "/7Xt78opBL1MfeJUhTBHy4jTk2og.jpg",
      "original_language": "en",
      "original_title": "Wolves at the Door",
      "genre_ids": [27, 53],
      "title": "Wolves at the Door",
      "vote_average": 4.7,
      "overview": "Four friends gather at an elegant home during the Summer of Love, 1969. Unbeknownst to them, deadly visitors are waiting outside. What begins as a simple farewell party turns to a night of primal terror as the intruders stalk and torment the four, who struggle for their lives against what appears to be a senseless attack.",
      "release_date": "2016-10-21"
    },
    {
      "popularity": 87.665,
      "vote_count": 4246,
      "video": false,
      "poster_path": "/y95lQLnuNKdPAzw9F9Ab8kJ80c3.jpg",
      "id": 38700,
      "adult": false,
      "backdrop_path": "/upUy2QhMZEmtypPW3PdieKLAHxh.jpg",
      "original_language": "en",
      "original_title": "Bad Boys for Life",
      "genre_ids": [28, 80, 53],
      "title": "Bad Boys for Life",
      "vote_average": 7.2,
      "overview": "Marcus and Mike are forced to confront new threats, career changes, and midlife crises as they join the newly created elite team AMMO of the Miami police department to take down the ruthless Armando Armas, the vicious leader of a Miami drug cartel.",
      "release_date": "2020-01-15"
    },
    {
      "popularity": 87.049,
      "id": 601844,
      "video": false,
      "vote_count": 17,
      "vote_average": 6.6,
      "title": "Becky",
      "release_date": "2020-06-05",
      "original_language": "en",
      "original_title": "Becky",
      "genre_ids": [53, 28],
      "backdrop_path": "/qTrpw2ZUvN7ywUu1kieEsvNDrgQ.jpg",
      "adult": false,
      "overview": "A teenager's weekend at a lake house with her father takes a turn for the worse when a group of convicts wreaks havoc on their lives.",
      "poster_path": "/xqbQtMffXwa3oprse4jiHBMBvdW.jpg"
    },
    {
      "popularity": 71.27,
      "vote_count": 14293,
      "video": false,
      "poster_path": "/c24sv2weTHPsmDa7jEMN0m2P3RT.jpg",
      "id": 315635,
      "adult": false,
      "backdrop_path": "/vc8bCGjdVp0UbMNLzHnHSLRbBWQ.jpg",
      "original_language": "en",
      "original_title": "Spider-Man: Homecoming",
      "genre_ids": [28, 12, 18, 878],
      "title": "Spider-Man: Homecoming",
      "vote_average": 7.4,
      "overview": "Following the events of Captain America: Civil War, Peter Parker, with the help of his mentor Tony Stark, tries to balance his life as an ordinary high school student in Queens, New York City, with fighting crime as his superhero alter ego Spider-Man as a new threat, the Vulture, emerges.",
      "release_date": "2017-07-05"
    },
    {
      "popularity": 70.082,
      "vote_count": 1358,
      "video": false,
      "poster_path": "/jtrhTYB7xSrJxR1vusu99nvnZ1g.jpg",
      "id": 522627,
      "adult": false,
      "backdrop_path": "/tintsaQ0WLzZsTMkTiqtMB3rfc8.jpg",
      "original_language": "en",
      "original_title": "The Gentlemen",
      "genre_ids": [28, 35, 80],
      "title": "The Gentlemen",
      "vote_average": 7.7,
      "overview": "American expat Mickey Pearson has built a highly profitable marijuana empire in London. When word gets out that he’s looking to cash out of the business forever it triggers plots, schemes, bribery and blackmail in an attempt to steal his domain out from under him.",
      "release_date": "2020-01-01"
    },
    {
      "popularity": 74.705,
      "vote_count": 12977,
      "video": false,
      "poster_path": "/udDclJoHjfjb8Ekgsd4FDteOkCU.jpg",
      "id": 475557,
      "adult": false,
      "backdrop_path": "/f5F4cRhQdUbyVbB5lTNCwUzD6BP.jpg",
      "original_language": "en",
      "original_title": "Joker",
      "genre_ids": [80, 18, 53],
      "title": "Joker",
      "vote_average": 8.2,
      "overview": "During the 1980s, a failed stand-up comedian is driven insane and turns to a life of crime and chaos in Gotham City while becoming an infamous psychopathic crime figure.",
      "release_date": "2019-10-02"
    },
    {
      "popularity": 108.265,
      "vote_count": 28,
      "video": false,
      "poster_path": "/ygCQnDEqUEIamBpdQdDYnFfxvgM.jpg",
      "id": 339095,
      "adult": false,
      "backdrop_path": "/t93doi7EzoqLFckidrGGnukFPwd.jpg",
      "original_language": "en",
      "original_title": "The Last Days of American Crime",
      "genre_ids": [28, 80, 53],
      "title": "The Last Days of American Crime",
      "vote_average": 5.2,
      "overview": "In the not-too-distant future, where as a final response to crime and terrorism, the U.S. government plans to broadcast a signal that will make it impossible for anyone to knowingly break the law.",
      "release_date": "2020-06-05"
    }
  ]
}
