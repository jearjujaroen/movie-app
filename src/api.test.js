import { fetchMovieImageConfigs, fetchMovies } from './api';
import movieImgConfigs from './testMocks/movieImgConfigs.json';
import movies from './testMocks/movies.json';

describe('it fetches movie configurations', () => {
  test('fetches successfully', () => {
    global.fetch = jest.fn(() =>
      Promise.resolve({
        json: () => Promise.resolve(movieImgConfigs),
      })
    );

    fetchMovieImageConfigs().then(data => expect(data).toMatchSnapshot());
  });
});

describe('it fetches popular movies', () => {
  test('fetches successfully', () => {
    global.fetch = jest.fn(() =>
      Promise.resolve({
        json: () => Promise.resolve(movies),
      })
    );

    fetchMovies().then(data => expect(data).toMatchSnapshot());
  });
});
