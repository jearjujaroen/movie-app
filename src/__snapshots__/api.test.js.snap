// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it fetches movie configurations fetches successfully 1`] = `
Object {
  "baseBackdropUrl": "http://image.tmdb.org/t/p/w1280",
  "basePosterUrl": "http://image.tmdb.org/t/p/original",
  "baseProfileUrl": "http://image.tmdb.org/t/p/original",
}
`;

exports[`it fetches popular movies fetches successfully 1`] = `
Array [
  Object {
    "adult": false,
    "backdrop_path": "/5BwqwxMEjeFtdknRV792Svo0K1v.jpg",
    "genre_ids": Array [
      18,
      878,
    ],
    "id": 419704,
    "original_language": "en",
    "original_title": "Ad Astra",
    "overview": "The near future, a time when both hope and hardships drive humanity to look to the stars and beyond. While a mysterious phenomenon menaces to destroy life on planet Earth, astronaut Roy McBride undertakes a mission across the immensity of space and its many perils to uncover the truth about a lost expedition that decades before boldly faced emptiness and silence in search of the unknown.",
    "popularity": 609.311,
    "poster_path": "/xBHvZcjRiWyobQ9kxBhO6B2dtRI.jpg",
    "release_date": "2019-09-17",
    "title": "Ad Astra",
    "video": false,
    "vote_average": 6,
    "vote_count": 3636,
  },
  Object {
    "adult": false,
    "backdrop_path": "/4phMeITqEnShF5C3ZD0Q8YQFMAB.jpg",
    "genre_ids": Array [
      28,
      878,
      53,
    ],
    "id": 574982,
    "original_language": "ru",
    "original_title": "The Blackout",
    "overview": "No meteorites hit Earth, no terrorists put the world in danger, no atomic war was started, but something did go wrong. Contact between most towns on Earth has been severed. A small ring-like area in Eastern Europe still has electricity, and maybe even life is being reported from the Space. What military forces find outside the Ring is shocking. There are dead corpses everywhere: in stores, in cars, on roads, in hospitals and railway stations. Who or what is destroying all life on Earth? How long will the last outpost of mankind survive?",
    "popularity": 416.549,
    "poster_path": "/avl9MEQhtvokNnzoWepkmHBZ2ss.jpg",
    "release_date": "2019-11-21",
    "title": "The Blackout",
    "video": false,
    "vote_average": 6,
    "vote_count": 31,
  },
  Object {
    "adult": false,
    "backdrop_path": "/ibMSQUtltvnmsOSrCsBrDNThzCu.jpg",
    "genre_ids": Array [
      28,
      35,
      878,
      10751,
    ],
    "id": 454626,
    "original_language": "en",
    "original_title": "Sonic the Hedgehog",
    "overview": "Based on the global blockbuster videogame franchise from Sega, Sonic the Hedgehog tells the story of the world’s speediest hedgehog as he embraces his new home on Earth. In this live-action adventure comedy, Sonic and his new best friend team up to defend the planet from the evil genius Dr. Robotnik and his plans for world domination.",
    "popularity": 163.863,
    "poster_path": "/aQvJ5WPzZgYVDrxLX4R6cLJCEaQ.jpg",
    "release_date": "2020-02-12",
    "title": "Sonic the Hedgehog",
    "video": false,
    "vote_average": 7.5,
    "vote_count": 4439,
  },
  Object {
    "adult": false,
    "backdrop_path": "/7Xt78opBL1MfeJUhTBHy4jTk2og.jpg",
    "genre_ids": Array [
      27,
      53,
    ],
    "id": 363111,
    "original_language": "en",
    "original_title": "Wolves at the Door",
    "overview": "Four friends gather at an elegant home during the Summer of Love, 1969. Unbeknownst to them, deadly visitors are waiting outside. What begins as a simple farewell party turns to a night of primal terror as the intruders stalk and torment the four, who struggle for their lives against what appears to be a senseless attack.",
    "popularity": 125.121,
    "poster_path": "/kizBwSEfvJFEOuYfwbOGtWZit3v.jpg",
    "release_date": "2016-10-21",
    "title": "Wolves at the Door",
    "video": false,
    "vote_average": 4.7,
    "vote_count": 120,
  },
  Object {
    "adult": false,
    "backdrop_path": "/ApiBzeaa95TNYliSbQ8pJv4Fje7.jpg",
    "genre_ids": Array [
      35,
      18,
      53,
    ],
    "id": 496243,
    "original_language": "ko",
    "original_title": "기생충",
    "overview": "All unemployed, Ki-taek's family takes peculiar interest in the wealthy and glamorous Parks for their livelihood until they get entangled in an unexpected incident.",
    "popularity": 124.024,
    "poster_path": "/7IiTTgloJzvGI1TAYymCfbfl3vT.jpg",
    "release_date": "2019-05-30",
    "title": "Parasite",
    "video": false,
    "vote_average": 8.5,
    "vote_count": 7772,
  },
  Object {
    "adult": false,
    "backdrop_path": "/ocUrMYbdjknu2TwzMHKT9PBBQRw.jpg",
    "genre_ids": Array [
      28,
      878,
    ],
    "id": 338762,
    "original_language": "en",
    "original_title": "Bloodshot",
    "overview": "After he and his wife are murdered, marine Ray Garrison is resurrected by a team of scientists. Enhanced with nanotechnology, he becomes a superhuman, biotech killing machine—'Bloodshot'. As Ray first trains with fellow super-soldiers, he cannot recall anything from his former life. But when his memories flood back and he remembers the man that killed both him and his wife, he breaks out of the facility to get revenge, only to discover that there's more to the conspiracy than he thought.",
    "popularity": 122.055,
    "poster_path": "/8WUVHemHFH2ZIP6NWkwlHWsyrEL.jpg",
    "release_date": "2020-03-05",
    "title": "Bloodshot",
    "video": false,
    "vote_average": 7,
    "vote_count": 2537,
  },
  Object {
    "adult": false,
    "backdrop_path": "/t93doi7EzoqLFckidrGGnukFPwd.jpg",
    "genre_ids": Array [
      28,
      80,
      53,
    ],
    "id": 339095,
    "original_language": "en",
    "original_title": "The Last Days of American Crime",
    "overview": "In the not-too-distant future, where as a final response to crime and terrorism, the U.S. government plans to broadcast a signal that will make it impossible for anyone to knowingly break the law.",
    "popularity": 108.265,
    "poster_path": "/ygCQnDEqUEIamBpdQdDYnFfxvgM.jpg",
    "release_date": "2020-06-05",
    "title": "The Last Days of American Crime",
    "video": false,
    "vote_average": 5.2,
    "vote_count": 28,
  },
  Object {
    "adult": false,
    "backdrop_path": "/5myQbDzw3l8K9yofUXRJ4UTVgam.jpg",
    "genre_ids": Array [
      28,
      12,
      878,
    ],
    "id": 429617,
    "original_language": "en",
    "original_title": "Spider-Man: Far from Home",
    "overview": "Peter Parker and his friends go on a summer trip to Europe. However, they will hardly be able to rest - Peter will have to agree to help Nick Fury uncover the mystery of creatures that cause natural disasters and destruction throughout the continent.",
    "popularity": 101.453,
    "poster_path": "/4q2NNj4S5dG2RLF9CpXsej7yXl.jpg",
    "release_date": "2019-06-28",
    "title": "Spider-Man: Far from Home",
    "video": false,
    "vote_average": 7.6,
    "vote_count": 7483,
  },
  Object {
    "adult": false,
    "backdrop_path": "/naXUDz0VGK7aaPlEpsuYW8kNVsr.jpg",
    "genre_ids": Array [
      28,
      27,
      53,
    ],
    "id": 514847,
    "original_language": "en",
    "original_title": "The Hunt",
    "overview": "Twelve strangers wake up in a clearing. They don't know where they are—or how they got there. In the shadow of a dark internet conspiracy theory, ruthless elitists gather at a remote location to hunt humans for sport. But their master plan is about to be derailed when one of the hunted turns the tables on her pursuers.",
    "popularity": 100.849,
    "poster_path": "/wxPhn4ef1EAo5njxwBkAEVrlJJG.jpg",
    "release_date": "2020-03-11",
    "title": "The Hunt",
    "video": false,
    "vote_average": 6.7,
    "vote_count": 794,
  },
  Object {
    "adult": false,
    "backdrop_path": "/xFxk4vnirOtUxpOEWgA1MCRfy6J.jpg",
    "genre_ids": Array [
      12,
      16,
      35,
      14,
      10751,
    ],
    "id": 508439,
    "original_language": "en",
    "original_title": "Onward",
    "overview": "In a suburban fantasy world, two teenage elf brothers embark on an extraordinary quest to discover if there is still a little magic left out there.",
    "popularity": 98.693,
    "poster_path": "/f4aul3FyD3jv3v4bul1IrkWZvzq.jpg",
    "release_date": "2020-02-29",
    "title": "Onward",
    "video": false,
    "vote_average": 7.9,
    "vote_count": 2105,
  },
  Object {
    "adult": false,
    "backdrop_path": "/cqa3sa4c4jevgnEJwq3CMF8UfTG.jpg",
    "genre_ids": Array [
      28,
      18,
      36,
      10752,
    ],
    "id": 530915,
    "original_language": "en",
    "original_title": "1917",
    "overview": "At the height of the First World War, two young British soldiers must cross enemy territory and deliver a message that will stop a deadly attack on hundreds of soldiers.",
    "popularity": 98.403,
    "poster_path": "/iZf0KyrE25z1sage4SYFLCCrMi9.jpg",
    "release_date": "2019-12-25",
    "title": "1917",
    "video": false,
    "vote_average": 7.9,
    "vote_count": 5277,
  },
  Object {
    "adult": false,
    "backdrop_path": "/kvbbK2rLGSJh9rf6gg1i1iVLYQS.jpg",
    "genre_ids": Array [
      28,
      35,
      80,
    ],
    "id": 495764,
    "original_language": "en",
    "original_title": "Birds of Prey (and the Fantabulous Emancipation of One Harley Quinn)",
    "overview": "Harley Quinn joins forces with a singer, an assassin and a police detective to help a young girl who had a hit placed on her after she stole a rare diamond from a crime lord.",
    "popularity": 98.069,
    "poster_path": "/h4VB6m0RwcicVEZvzftYZyKXs6K.jpg",
    "release_date": "2020-02-05",
    "title": "Birds of Prey (and the Fantabulous Emancipation of One Harley Quinn)",
    "video": false,
    "vote_average": 7.2,
    "vote_count": 4342,
  },
  Object {
    "adult": false,
    "backdrop_path": "/sd4xN5xi8tKRPrJOWwNiZEile7f.jpg",
    "genre_ids": Array [
      12,
      16,
      35,
      10751,
    ],
    "id": 920,
    "original_language": "en",
    "original_title": "Cars",
    "overview": "Lightning McQueen, a hotshot rookie race car driven to succeed, discovers that life is about the journey, not the finish line, when he finds himself unexpectedly detoured in the sleepy Route 66 town of Radiator Springs. On route across the country to the big Piston Cup Championship in California to compete against two seasoned pros, McQueen gets to know the town's offbeat characters.",
    "popularity": 93.441,
    "poster_path": "/qa6HCwP4Z15l3hpsASz3auugEW6.jpg",
    "release_date": "2006-06-08",
    "title": "Cars",
    "video": false,
    "vote_average": 6.8,
    "vote_count": 9456,
  },
  Object {
    "adult": false,
    "backdrop_path": "/lvtEcdBmmrxzAQvRIYyDRQDeDJM.jpg",
    "genre_ids": Array [
      27,
      37,
    ],
    "id": 610201,
    "original_language": "en",
    "original_title": "The Pale Door",
    "overview": "After a stagecoach robbery goes bad, two brothers leading a gang of cowboys must survive the night in a ghost town inhabited by a coven of witches.",
    "popularity": 92.519,
    "poster_path": "/shx7pGYQ9s4t7WFe19eES2WOrMS.jpg",
    "release_date": "2020-06-18",
    "title": "The Pale Door",
    "video": false,
    "vote_average": 0,
    "vote_count": 0,
  },
  Object {
    "adult": false,
    "backdrop_path": "/upUy2QhMZEmtypPW3PdieKLAHxh.jpg",
    "genre_ids": Array [
      28,
      80,
      53,
    ],
    "id": 38700,
    "original_language": "en",
    "original_title": "Bad Boys for Life",
    "overview": "Marcus and Mike are forced to confront new threats, career changes, and midlife crises as they join the newly created elite team AMMO of the Miami police department to take down the ruthless Armando Armas, the vicious leader of a Miami drug cartel.",
    "popularity": 87.665,
    "poster_path": "/y95lQLnuNKdPAzw9F9Ab8kJ80c3.jpg",
    "release_date": "2020-01-15",
    "title": "Bad Boys for Life",
    "video": false,
    "vote_average": 7.2,
    "vote_count": 4246,
  },
  Object {
    "adult": false,
    "backdrop_path": "/qTrpw2ZUvN7ywUu1kieEsvNDrgQ.jpg",
    "genre_ids": Array [
      53,
      28,
    ],
    "id": 601844,
    "original_language": "en",
    "original_title": "Becky",
    "overview": "A teenager's weekend at a lake house with her father takes a turn for the worse when a group of convicts wreaks havoc on their lives.",
    "popularity": 87.049,
    "poster_path": "/xqbQtMffXwa3oprse4jiHBMBvdW.jpg",
    "release_date": "2020-06-05",
    "title": "Becky",
    "video": false,
    "vote_average": 6.6,
    "vote_count": 17,
  },
  Object {
    "adult": false,
    "backdrop_path": "/fKtYXUhX5fxMxzQfyUcQW9Shik6.jpg",
    "genre_ids": Array [
      12,
      16,
      35,
      9648,
      10751,
    ],
    "id": 385103,
    "original_language": "en",
    "original_title": "Scoob!",
    "overview": "In Scooby-Doo’s greatest adventure yet, see the never-before told story of how lifelong friends Scooby and Shaggy first met and how they joined forces with young detectives Fred, Velma, and Daphne to form the famous Mystery Inc. Now, with hundreds of cases solved, Scooby and the gang face their biggest, toughest mystery ever: an evil plot to unleash the ghost dog Cerberus upon the world. As they race to stop this global “dogpocalypse,” the gang discovers that Scooby has a secret legacy and an epic destiny greater than anyone ever imagined.",
    "popularity": 79.754,
    "poster_path": "/jHo2M1OiH9Re33jYtUQdfzPeUkx.jpg",
    "release_date": "2020-05-15",
    "title": "Scoob!",
    "video": false,
    "vote_average": 8,
    "vote_count": 906,
  },
  Object {
    "adult": false,
    "backdrop_path": "/f5F4cRhQdUbyVbB5lTNCwUzD6BP.jpg",
    "genre_ids": Array [
      80,
      18,
      53,
    ],
    "id": 475557,
    "original_language": "en",
    "original_title": "Joker",
    "overview": "During the 1980s, a failed stand-up comedian is driven insane and turns to a life of crime and chaos in Gotham City while becoming an infamous psychopathic crime figure.",
    "popularity": 74.705,
    "poster_path": "/udDclJoHjfjb8Ekgsd4FDteOkCU.jpg",
    "release_date": "2019-10-02",
    "title": "Joker",
    "video": false,
    "vote_average": 8.2,
    "vote_count": 12977,
  },
  Object {
    "adult": false,
    "backdrop_path": "/vc8bCGjdVp0UbMNLzHnHSLRbBWQ.jpg",
    "genre_ids": Array [
      28,
      12,
      18,
      878,
    ],
    "id": 315635,
    "original_language": "en",
    "original_title": "Spider-Man: Homecoming",
    "overview": "Following the events of Captain America: Civil War, Peter Parker, with the help of his mentor Tony Stark, tries to balance his life as an ordinary high school student in Queens, New York City, with fighting crime as his superhero alter ego Spider-Man as a new threat, the Vulture, emerges.",
    "popularity": 71.27,
    "poster_path": "/c24sv2weTHPsmDa7jEMN0m2P3RT.jpg",
    "release_date": "2017-07-05",
    "title": "Spider-Man: Homecoming",
    "video": false,
    "vote_average": 7.4,
    "vote_count": 14293,
  },
  Object {
    "adult": false,
    "backdrop_path": "/tintsaQ0WLzZsTMkTiqtMB3rfc8.jpg",
    "genre_ids": Array [
      28,
      35,
      80,
    ],
    "id": 522627,
    "original_language": "en",
    "original_title": "The Gentlemen",
    "overview": "American expat Mickey Pearson has built a highly profitable marijuana empire in London. When word gets out that he’s looking to cash out of the business forever it triggers plots, schemes, bribery and blackmail in an attempt to steal his domain out from under him.",
    "popularity": 70.082,
    "poster_path": "/jtrhTYB7xSrJxR1vusu99nvnZ1g.jpg",
    "release_date": "2020-01-01",
    "title": "The Gentlemen",
    "video": false,
    "vote_average": 7.7,
    "vote_count": 1358,
  },
]
`;
